actions/admin/site/update_advanced.php

```diff
@@ -59,34 +59,34 @@
 	}
 }
 
-set_config('simplecache_minify_js', 'on' === get_input('simplecache_minify_js'), $site->getGUID());
-set_config('simplecache_minify_css', 'on' === get_input('simplecache_minify_css'), $site->getGUID());
+elgg_save_config('simplecache_minify_js', 'on' === get_input('simplecache_minify_js'));
+elgg_save_config('simplecache_minify_css', 'on' === get_input('simplecache_minify_css'));
 
 if ('on' === get_input('system_cache_enabled')) {
 	elgg_enable_system_cache();
 } else {
 	elgg_disable_system_cache();
 }
 
-set_config('default_access', get_input('default_access', ACCESS_PRIVATE), $site->getGUID());
+elgg_save_config('default_access', get_input('default_access', ACCESS_PRIVATE));
```

Comment:
maybe (int) get_input, otherwise a string is stored and this complicates compares